amundsen_application/static/js/config/config-utils.ts

```diff
@@ -84,7 +84,7 @@ export function getBadgeConfig(badgeName: string): BadgeStyleConfig {
 
   return {
     style: BadgeStyle.DEFAULT,
-    displayName: badgeName.replace(/[-_]/g, ' '),
+    displayName: badgeName,
```

Comment:
@ttannis do you know why we did `badgeName.replace(/[-_]/g, ' '),` before?

Reply:
I think product/design wanted the text in the UI to appear differently than the badge name stored in metadata. But we don't see value in that anymore since badge search requires knowing the exact name. If we modify the badge name in the UI users aren't going to know what they need to type in when they want to use the filter UI.

Alternatively though, if we also create the API to get all the badges -- we could also go ahead and update the badge filter to UI so users can select instead of having to know what name to type in.